Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs

```diff
@@ -7,15 +7,15 @@
 
 namespace Blizzard.IO.RabbitMQ.Rpc
 {
-    public class NetqRabbitRpcConnection : INetqRpcRabbitConnection<Func<Type, object>>
+    public class NetqRabbitRpcConnection : INetqRpcRabbitConnection<Func<Type, object>>, IDisposable
```

Comment:
I'm not sure we want this `IDisposable`. We want this class to be handled by the `GC`

Reply:
we talked about it